standards/documentation.md

```diff
@@ -25,6 +25,9 @@ README documents MUST contain the following sections:
 ### Optional README sections/additions
 
 - SHOULD include current, stable version information
+- SHOULD describe git workflow, including if branches map to certain environments.  
+  As an example, see [NYPL/scsb_item_updater's 'Git Workflow & Deployment Section'](https://github.com/NYPL-discovery/scsb_item_updater#git-workflow--deployment)
+- SHOULD instructions about deployment.
```

Comment:
I think you're missing a word here? `include`?

Reply:
Fixed! Thanks 🦅 👁 !